libraries/AP_HAL_ChibiOS/hwdef/scripts/defaults_bootloader.h

```diff
@@ -35,3 +35,30 @@
 #ifndef AP_WATCHDOG_SAVE_FAULT_ENABLED
 #define AP_WATCHDOG_SAVE_FAULT_ENABLED 0
 #endif
+
+// less LWIP functionality in the bootloader
+#define LWIP_DHCP 0
```

Comment:
```suggestion
#define LWIP_DHCP AP_BOOTLOADER_NETWORK_USE_DHCP
```

Reply:
no, I don't think we want DHCP in the bootloader